### ISSUE TITLE
Tests fail on http-cookie / ruby 3.0

### ISSUE THREAD

**Issue by bastelfreak** [sparklemotion/http-cookie#38]
Hi!
I'm trying to package http-cookie for arch linux. During `rake test --trace` I get the following error:

```
/usr/bin/ruby -w -I"lib" /usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/rake_test_loader.rb "test/test_http_cookie.rb" "test/test_http_cookie_jar.rb"
/build/ruby-http-cookie/src/http-cookie-1.0.4/test/test_http_cookie.rb:752: warning: ambiguous first argument; put parentheses or a space even after `-' operator
Loaded suite /usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/rake_test_loader
Started
................................................E
===============================================================================
Error: test_yaml_expires(TestHTTPCookie): Psych::DisallowedClass: Tried to load unspecified class: HTTP::Cookie
/usr/lib/ruby/gems/3.0.0/gems/psych-4.0.3/lib/psych/class_loader.rb:99:in `find'
/usr/lib/ruby/gems/3.0.0/gems/psych-4.0.3/lib/psych/class_loader.rb:28:in `load'
/usr/lib/ruby/gems/3.0.0/gems/psych-4.0.3/lib/psych/visitors/to_ruby.rb:424:in `resolve_class'
/usr/lib/ruby/gems/3.0.0/gems/psych-4.0.3/lib/psych/visitors/to_ruby.rb:213:in `visit_Psych_Nodes_Mapping'
/usr/lib/ruby/gems/3.0.0/gems/psych-4.0.3/lib/psych/visitors/visitor.rb:30:in `visit'
/usr/lib/ruby/gems/3.0.0/gems/psych-4.0.3/lib/psych/visitors/visitor.rb:6:in `accept'
/usr/lib/ruby/gems/3.0.0/gems/psych-4.0.3/lib/psych/visitors/to_ruby.rb:35:in `accept'
/usr/lib/ruby/gems/3.0.0/gems/psych-4.0.3/lib/psych/visitors/to_ruby.rb:318:in `visit_Psych_Nodes_Document'
/usr/lib/ruby/gems/3.0.0/gems/psych-4.0.3/lib/psych/visitors/visitor.rb:30:in `visit'
/usr/lib/ruby/gems/3.0.0/gems/psych-4.0.3/lib/psych/visitors/visitor.rb:6:in `accept'
/usr/lib/ruby/gems/3.0.0/gems/psych-4.0.3/lib/psych/visitors/to_ruby.rb:35:in `accept'
/usr/lib/ruby/gems/3.0.0/gems/psych-4.0.3/lib/psych.rb:335:in `safe_load'
/usr/lib/ruby/gems/3.0.0/gems/psych-4.0.3/lib/psych.rb:370:in `load'
/build/ruby-http-cookie/src/http-cookie-1.0.4/test/test_http_cookie.rb:1083:in `test_yaml_expires'
     1080:     assert_equal false, cookie.session?
     1081:     assert_equal nil, cookie.max_age
     1082:
  => 1083:     ycookie = YAML.load(cookie.to_yaml)
     1084:     assert_equal false, ycookie.session?
     1085:     assert_equal nil, ycookie.max_age
     1086:     assert_in_delta cookie.expires, ycookie.expires, 1
===============================================================================
```

With the following trace:

```
Command failed with status (1): [ruby -w -I"lib" /usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/rake_test_loader.rb "test/test_http_cookie.rb" "test/test_http_cookie_jar.rb" ]
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/testtask.rb:130:in `block (3 levels) in define'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/file_utils.rb:57:in `sh'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/file_utils.rb:104:in `ruby'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/testtask.rb:117:in `block (2 levels) in define'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/file_utils_ext.rb:58:in `verbose'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/testtask.rb:111:in `block in define'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/task.rb:281:in `block in execute'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/task.rb:281:in `each'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/task.rb:281:in `execute'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/task.rb:219:in `block in invoke_with_call_chain'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/task.rb:199:in `synchronize'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/task.rb:199:in `invoke_with_call_chain'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/task.rb:188:in `invoke'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/application.rb:160:in `invoke_task'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/application.rb:116:in `block (2 levels) in top_level'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/application.rb:116:in `each'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/application.rb:116:in `block in top_level'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/application.rb:125:in `run_with_threads'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/application.rb:110:in `top_level'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/application.rb:83:in `block in run'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/application.rb:186:in `standard_exception_handling'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/application.rb:80:in `run'
/usr/lib/ruby/gems/3.0.0/gems/rake-13.0.6/exe/rake:27:in `<top (required)>'
/usr/bin/rake:23:in `load'
/usr/bin/rake:23:in `<main>'
Tasks: TOP => test
```

Do you have any clue what's broken here?

**Comment by geoffyoungs** [comment]
Looks like this was fixed here: d12449a983d3dd660c5fe1f2b135c35e83755cc3

**Comment by rogerluan** [comment]
Thanks for pointing it out @geoffyoungs 🙏
